crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -356,6 +356,23 @@ impl Room {
         self.inner.read().unwrap().latest_event.clone()
     }
 
+    /// Return the most recent few encrypted events. When the keys come through
+    /// to decrypt these, the most recent relevant one will replace
+    /// latest_event. (We can't tell which one is relevant until
+    /// they are decrypted.)
+    pub fn latest_encrypted_events(&self) -> RingBuffer<Raw<AnySyncTimelineEvent>> {
+        // TODO: clone feels quite painful here
+        self.inner.read().unwrap().latest_encrypted_events.clone()
+    }
+
+    /// Replace our latest_event with the supplied event, and delete it and all
+    /// older encrypted events from latest_encrypted_events, given that the
+    /// new event was at the supplied index in the latest_encrypted_events
+    /// list.
+    pub fn latest_event_decrypted(&mut self, event: SyncTimelineEvent, index: usize) {
```

Comment:
```diff
- pub fn
+ fn
```

Is it worth having it in the public API?

Reply:
Changed to `pub(crate)`